messages/commonFlags.md

```diff
@@ -60,6 +60,14 @@ The command immediately returns the job ID and control of the terminal to you. T
 
 Run the command asynchronously.
 
+# promote.wait.description
+
+If the command continues to run after the wait period, the CLI returns control of the terminal window to you. To resume watching the deploy operation, run this command again. To check the status of the deploy operation, run "sf deploy pipeline report".
```

Comment:
This message is a bit different that the approved UIT. Did you check it with Emily? If so, please have the UIT updated.

Reply:
Where is the approved UIT ?? This https://salesforce.quip.com/fxbMA99bbKIq ?